doc/manual/manual.tex

```diff
@@ -6630,9 +6630,12 @@ \subsubsection{Reading in compositional initial composition files generated with
 \label{sec:geomio}
 \textit{This section was contributed by Juliane Dannberg}
 
+\note{This cookbook is based on a developer version of geomIO from July 2016. In the meantime, the development of 
+geomIO continued, and there is now a publication\cite{bauvillegeomio} that describes its features and how they can be used in more detail.}
+
 Many geophysical setups require initial conditions with several different materials and complex geometries. 
 Hence, sometimes it would be easier to generate the initial geometries of the materials as a drawing instead 
-of by writing code. The MATLAB-based library geomIO (\url{http://geomio.bitbucket.org/}) provides a convenient tool
+of by writing code. The MATLAB-based library geomIO \cite[\url{https://bitbucket.org/geomio/geomio}]{bauvillegeomio} provides a convenient tool
```

Comment:
This is a weird way to cite. Why do you do it this way? Would you mind leaving the URL in the text as before (you can also add the citation entry).

Reply:
Sure. Fixed.